Collapsible/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react'
+import {Motion, spring} from 'react-motion'
+
+export default class Collapsible extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      height: 0
+    }
+  }
+
+  componentDidMount () {
+    this.setState({
+      height: getHeight(this.content)
+    })
+
+    window.addEventListener('resize', () => {
+      const height = getHeight(this.content)
+
+      if (this.state.height !== height) {
+        this.setState({ height })
```

Comment:
I don't think we have time to fix this before the current deadline. I'll move your comment to it's own issue since I guess this is not a showstopper.